client/components/language-picker/language-names.js

```diff
@@ -0,0 +1,141 @@
+	'la': translate => translate( 'Latin' ),
+	'li': translate => translate( 'Limburgish' ),
+	'lo': translate => translate( 'Lao' ),
+	'lt': translate => translate( 'Lithuanian' ),
+	'lv': translate => translate( 'Latvian' ),
+	'mk': translate => translate( 'Macedonian' ),
+	'ml': translate => translate( 'Malayalam' ),
+	'mn': translate => translate( 'Mongolian' ),
+	'mr': translate => translate( 'Marathi' ),
+	'ms': translate => translate( 'Malay' ),
+	'mt': translate => translate( 'Maltese' ),
+	'mwl': translate => translate( 'Mirandese' ),
+	'nah': translate => translate( 'Nahuatl' ),
+	'nap': translate => translate( 'Neapolitan' ),
+	'nds': translate => translate( 'Low German' ),
+	'ne': translate => translate( 'Nepali' ),
+	'nl': translate => translate( 'Dutch' ),
+	'nn': translate => translate( 'Norwegian Nynorsk' ),
+	'no': translate => translate( 'Norwegian' ),
+	'non': translate => translate( 'Old Norse' ),
+	'nv': translate => translate( 'Navajo' ),
+	'oci': translate => translate( 'Occitan' ),
+	'or': translate => translate( 'Odia' ),
+	'os': translate => translate( 'Ossetic' ),
+	'pa': translate => translate( 'Punjabi' ),
+	'pl': translate => translate( 'Polish' ),
+	'ps': translate => translate( 'Pashto' ),
+	'pt': translate => translate( 'Portuguese' ),
+	'pt-br': translate => translate( 'Brazilian Portuguese' ),
+	'qu': translate => translate( 'Quechua' ),
+	'ro': translate => translate( 'Romanian' ),
+	'ru': translate => translate( 'Russian' ),
+	'rup': translate => translate( 'Aromanian' ),
+	'sc': translate => translate( 'Sardinian' ),
+	'si': translate => translate( 'Sinhala' ),
+	'sk': translate => translate( 'Slovak' ),
+	'sl': translate => translate( 'Slovenian' ),
+	'snd': translate => translate( 'Sindhi' ),
+	'so': translate => translate( 'Somali' ),
+	'sq': translate => translate( 'Albanian' ),
+	'sr': translate => translate( 'Serbian' ),
+	'su': translate => translate( 'Sundanese' ),
+	'sv': translate => translate( 'Swedish' ),
+	'ta': translate => translate( 'Tamil' ),
+	'te': translate => translate( 'Telugu' ),
+	'th': translate => translate( 'Thai' ),
+	'tir': translate => translate( 'Tigrinya' ),
+	'tl': translate => translate( 'Tagalog' ),
+	'tr': translate => translate( 'Turkish' ),
+	'tt': translate => translate( 'Tatar' ),
+	'ty': translate => translate( 'Tahitian' ),
+	'udm': translate => translate( 'Udmurt' ),
+	'ug': translate => translate( 'Uyghur' ),
+	'uk': translate => translate( 'Ukrainian' ),
+	'ur': translate => translate( 'Urdu' ),
+	'uz': translate => translate( 'Uzbek' ),
+	'vec': translate => translate( 'Venetian' ),
+	'vi': translate => translate( 'Vietnamese' ),
+	'wa': translate => translate( 'Walloon' ),
+	'xal': translate => translate( 'Kalmyk' ),
+	'yi': translate => translate( 'Yiddish' ),
+	'yo': translate => translate( 'Yoruba' ),
+	'za': translate => translate( 'Zhuang' ),
+	'zh-cn': translate => translate( 'Simplified Chinese' ),
+	'zh-tw': translate => translate( 'Traditional Chinese' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Traditional', { context: 'Theme Showcase term name'} )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/language-picker-english-lang-names)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).